_partners/xoriant.md

```diff
@@ -0,0 +1,70 @@
+---
+# Partner name used on the partners collection page (/partners).
+name: Xoriant
+
+# name_long is used on the extended partner details page.
+name_long: 'Xoriant'
+
+# The partner logo is used on the partners collection page (/partners).
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/xoriant.png'
+
+# URL to the partner's website.
+link: 'https://www.xoriant.com'
+
+# logo_large is used on the extended partner details page.
+# It is normative to upload all extended partner page images to a subdirectory of /assets/media/partners/ with the name of the partner as the directory name.
+# For example given a partner named "SampleCompany" the logo_large image would be uploaded to /assets/media/partners/samplecompany/logo_large.png
+logo_large: '/assets/media/partners/xoriant/xoriant.png'
+
+# Large image displayed in the top right of the extended partner details page.
+product_image: '/assets/media/partners/xoriant/xoriant.png'
+# Partner company description used on the extended partner details page.
+description: Xoriant is a global digital engineering services company, enabling innovation & accelerating time to market for clients by building digital products and platforms. We are driven by our product engineering DNA, culture of innovation, and technology capabilities in Cloud, Security, Data & AI.
+
+
+# Comma delimited list of business types used on the extended partner details page in the side panel.
+business_type: OpenSearch Services
+
+# Comma delimited list of geographic regions used on the extended partner details page in the side panel.
+region: North America, Europe, Asia
+
+# Comma delimited list of one or more of email addresses, phone numbers, and web URLs that can be used to contat the partner.
+contact: info@xoriant.com
+
+# Comma delimited list of one or more feature areas of the OpenSearch platform that the partner specializes in.
+opensearch_tech:  Search, Analytics, Observability, Migration
+
+# Comma delimited list of one or more industries that the partner specializes in serving.
+industries:  Manufacturing and Automotive, Supply Chain, Consumer Services, Education, Energy and Utilities, Financial Services, Healthcare, Media and Entertainment, Public sector, Retail and e-commerce, Software and Technology
+
+# If the partner only has one office location then use the main_office_location key
+# otherwise use the multiple_office_locations key.
+# In the template rendering multiple_office_locations takes precedent,
+# and if both are used only the multiple_office_locations data will be rendered
+# in the template omitting the main_office_location data.
+main_office_location:
```

Comment:
@Ram-Kumaar-ML The format for the address requires a multiline string format with new lines preserved using the `|` character. Here's a SO discussion that gives some straight forward detail about the variants. https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines
Also, refer to `_partners/_sample.md`, or real world examples in many other `*.md` within the `_partners` collection.

Reply:
@astephanus - I'm still seeing that odd error when I pull it down locally - can you give it another look? 

"_partners/xoriant.md: (<unknown>): found character that cannot start any token while scanning for the next token at line 47 column 1"